CMakeLists.txt

```
@@ -99,9 +99,23 @@ include_directories(
   "${LIB_INCLUDES}"
 )
 
+set(executables "")
+
+function(add_mvc_executable name)
```

```suggestion
function(add_mvc_sources name)
```
The function is adding a list of source files, not executables.

---

I reused the same term as the basic CMake function `add_executable`, which also involves adding source files, although the name is "like that".

---

But the `add_executable` creates an executable with a list of sources, so it does something different.

`add_executable` here create the `file-share` executable with the sources `source/main.cpp, source/Application.cpp`. 

The add_mvc_x function does not create an executable, it just adds a list of sources to be included in the final executable